artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/TransportConstants.java

```diff
@@ -255,6 +255,10 @@
 
    public static final int DEFAULT_STOMP_MAX_FRAME_PAYLOAD_LENGTH = 65536;
 
+   public static final String NETTY_HANDSHAKE_TIMEOUT = "protocol-header-timeout";
```

Comment:
Names here should be consistent to avoid confusion between the code/config.  Maybe something like:

  HANDSHAKE_TIMEOUT = "handshake-timeout"